src/controllers/Trips.js

```diff
@@ -150,7 +150,84 @@ class Trips {
   * @returns {object} Details of booked trips
   * @memberof Trips
   */
-  static async return(req, res, next) {
+  static async returnTrip(req, res, next) {
+    try {
+      const { email } = req;
+      const aUser = await getUser(email);
+      if (!aUser) {
+        return serverResponse(res, 404, ...['error', 'message', `Cannot Find User With Email: ${email}`]);
+      }
+      const {
+        userId,
+        linemanager: lineManagerUser,
+        firstname: firstName,
+        lastname: lastName,
+        notifyemail: notifyEmail
+      } = aUser;
+      if (!lineManagerUser) {
+        return serverResponse(res, 400, ...['error', 'message', 'Line Manager must be present to continue']);
```

Comment:
This error message has been updated. can you check the one way trip method to see an example?